assets/src/edit-story/components/library/panes/text/textSets/raw/section_header.json

```diff
@@ -0,0 +1,2508 @@
+              "hDes": -500,
+              "lGap": 0
+            }
+          },
+          "fontSize": 18,
+          "backgroundColor": {
+            "color": {
+              "r": 196,
+              "g": 196,
+              "b": 196
+            }
+          },
+          "lineHeight": 1.5,
+          "textAlign": "center",
+          "padding": {
+            "locked": true,
+            "horizontal": 0,
+            "vertical": 0
+          },
+          "type": "text",
+          "content": "The aesthetic quality of a product is integral to its usefulness because products we use every day affect our person and our well-being. But only well-executed objects can be beautiful.",
+          "fontWeight": 400,
+          "width": 336,
+          "height": 100,
+          "scale": 100,
+          "focalX": 50,
+          "focalY": 50,
+          "basedOn": "9cde7138-1466-465e-b74c-76c5c3675e32",
+          "id": "4ea9d2da-f875-45aa-8ba1-d8032ed9df0b",
+          "x": 40,
+          "y": 466
+        },
+        {
+          "opacity": 100,
+          "flip": {
+            "vertical": false,
+            "horizontal": false
+          },
+          "rotationAngle": 0,
+          "lockAspectRatio": false,
+          "backgroundColor": {
+            "color": {
+              "r": 0,
+              "g": 0,
+              "b": 0
+            }
+          },
+          "type": "shape",
+          "x": 189,
+          "y": 369,
+          "width": 38,
+          "height": 1,
+          "scale": 100,
+          "focalX": 50,
+          "focalY": 50,
+          "mask": {
+            "type": "rectangle"
+          },
+          "id": "1026e4b9-721b-4d41-86f8-db853af66e04"
+        }
+      ],
+      "backgroundColor": {
+        "color": {
+          "r": 255,
+          "g": 255,
+          "b": 255
+        }
+      },
+      "type": "page",
+      "id": "e1d2a549-46ca-4f67-babc-efbfe8901406"
+    },
+    {
+      "backgroundOverlay": "none",
+      "elements": [
+        {
+          "opacity": 100,
+          "flip": {
+            "vertical": false,
+            "horizontal": false
+          },
+          "rotationAngle": 0,
+          "lockAspectRatio": true,
+          "backgroundColor": {
+            "color": {
+              "r": 255,
+              "g": 255,
+              "b": 255
+            }
+          },
+          "x": 1,
+          "y": 1,
+          "width": 1,
+          "height": 1,
+          "mask": {
+            "type": "rectangle"
+          },
+          "isBackground": true,
+          "isDefaultBackground": true,
+          "type": "shape",
+          "id": "a7fd86d3-8084-4712-a658-e4c82568e557"
+        },
+        {
+          "opacity": 100,
+          "flip": {
+            "vertical": false,
+            "horizontal": false
+          },
+          "rotationAngle": 0,
+          "lockAspectRatio": true,
+          "backgroundTextMode": "NONE",
+          "font": {
+            "family": "Poppins",
```

Comment:
I believe this font should be Merriweather not Poppins
(figma)
<img width="312" alt="Screen Shot 2020-09-18 at 9 42 30 AM" src="https://user-images.githubusercontent.com/10720454/93623327-6d734100-f993-11ea-9c87-3c518eb5c2d2.png">

(editor)
![Screen Shot 2020-09-18 at 9 42 44 AM](https://user-images.githubusercontent.com/10720454/93623330-6e0bd780-f993-11ea-9f28-b5be0e181041.png)